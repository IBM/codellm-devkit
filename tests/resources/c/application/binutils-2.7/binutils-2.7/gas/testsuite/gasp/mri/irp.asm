	irp	param,arg1,arg2,arg3
	dc.l	param
	endr
	end	quit
