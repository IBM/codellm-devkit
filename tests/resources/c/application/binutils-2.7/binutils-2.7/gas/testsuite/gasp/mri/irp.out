;	irp	param,arg1,arg2,arg3
;	dc.l	param
;	endr
	dc.l	arg1
	dc.l	arg2
	dc.l	arg3
;	end	quit
	end	quit
